.vimrc

```diff
@@ -68,14 +115,14 @@
 " }
 
 " Use before config if available {
-    if filereadable(expand("~/.vimrc.before"))
-        source ~/.vimrc.before
+    if filereadable(g:spf13_vimrc_before)
+        exec 'source ' . g:spf13_vimrc_before
```

Comment:
I couldn't find a way to git around this ugly exec 'source ' .  g:spf13_some_variable if I wanted to reference variables in my source call... Anyone know of a way? (VimL has to be one of the worst languages...)